crates/humanode-peer/src/chain_spec.rs

```diff
@@ -252,6 +252,81 @@ const EXISTENTIAL_DEPOSIT: Balance = 500;
 /// The initial pot accounts balance for testnet genesis.
 const INITIAL_POT_ACCOUNT_BALANCE: Balance = EXISTENTIAL_DEPOSIT + DEV_ACCOUNT_BALANCE;
 
+/// A helper function to prepare native and evm accounts.
+#[allow(clippy::type_complexity)]
+fn prepare_accounts_balances(
```

Comment:
I propose two changes here:
- this fn should take two lists of account/balance pair instead of taking four lists of accouts;
- this fn should return simple balances; the conversion to `fp_evm::GenesisAccount`s for instance should be kept outside of this fn
- instead of returning the whole list of accounts, this fn could return just a tuple of two balances - for native and evm bridge

What do you think?

Reply:
The first suggestions sounds helpful to avoid focusing on different balance values for different account types. The second and the third ones are outcome of the first one :)

Reply:
https://github.com/humanode-network/humanode/pull/700/commits/48e26183a49c09e248689abbc1de043d30359e80